README.md

```diff
@@ -61,7 +61,7 @@ gnome-keyring-yubikey-unlock/create_secret_file.sh /path/to/your_secret [Your Gn
 
 As an example, I need to input `login:My_Very_Long_Login_Password`. (You may use `seahorse` or `tools/list_keyrings.sh` to determine the name of your keyring)
 
-Then, add the following command to gnome-autostart. You should know how to auto-run a command after starting gnome.
+Then, add the following command to gnome-autostart. See [Start Via Systemd](doc/systemd.md).
```

Comment:
I suggest something like this: 

```
Then, add the following command to gnome-autostart. If you don't know how to do it, [read me](doc/how-to-gnome-autostart.md)! 
```

And in the `doc/how-to-gnome-autostart.md`, you need to tell the user how to autostart by systemd, and I will add content to tell user how to autostart by `./.config/autostart`. So the user could choose between these different methods.

Reply:
Sounds good. I updated the commit with your suggestion.

Reply:
Great! Thanks for your contribution~